modules/service/src/main/scala/lucuma/odb/graphql/input/sourceprofile/SourceProfileInput.scala

```diff
@@ -44,7 +44,7 @@ object SourceProfileInput {
       ) =>
         (rPoint, rUniform, rGaussian).parTupled.flatMap {
 
-          // If the user provides a full definitiom we can change the source profile
+          // If the user provides a full definition we can change the source profile
           case (Some(Left(p)), None, None) => Result(_ => Result(Point(p)))
```

Comment:
This case is matched by the new test case instead of the one we want, down at line 53.